Shared/Samples/Display map from mobile map package/DisplayMapFromMobileMapPackageView.swift

```diff
@@ -15,6 +15,7 @@
 import ArcGIS
 import SwiftUI
 
+@MainActor
```

Comment:
This sample can be refactored. No need to revert, will refactor in https://github.com/Esri/arcgis-maps-sdk-swift-samples/issues/456